index.html

```diff
@@ -172,12 +172,14 @@ <h3>let's connect</h3>
                     <h2 class="contact-header">Contact me</h2>
                     <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
                 </div>
-                <form method="post" action="https://formspree.io/f/xknedrqa" class="contact-form">
+                <form method="post" action="https://formspree.io/f/xknedrqa" class="contact-form" id="contact-form">
```

Comment:
- [ ] I really commend you for the great job you have done implementing validations to your form. its really working if I try to submit the form while a have my an uppercase letter in my email, it throws an error 👏 👏 👏 . But currently i noticed i could not submit my form normally when i have my the `input` form well written
> N.B:  If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

- I would kindly suggest you fix this before you resubmit for review